package.json

```diff
@@ -15,6 +15,9 @@
     "dev:mock": "NODE_ENV=development NX_ENABLE_MOCKS=1 nx run-many --target=dev --all",
     "clear-cache": "nx clear-cache"
   },
+  "exports": {
+    "./package.json": "./package.json"
+  },
```

Comment:
Why do you need to export the package.json?

Reply:
I always used it like this so I could use it as a module

Reply:
I don't see the point, though. In this case, it's not useful at all.

Reply:
```suggestion
```

Reply:
> 

acepted